### DISCUSSION TITLE
Relative reference for background PDF

### DISCUSSION THREAD

**Discussion by cquart** [xournalpp/xournalpp#4171] [Ideas]
When PDFs annotated in an .xopp file are moved to a different location, it can no longer locate the PDF and you have to show it the new location. This is annoying if you move the location of a folder containing many PDFs referenced by many .xopp files. It would be nice if there was an option for xournal++ to create references to pdfs in the same folder as the .xopp file so references aren't broken when the whole folder is moved to a different location.

**Comment by hyperupcall** [comment]
This is being solved in #4262
